features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -789,9 +958,48 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
         );
     };
 
+    const resolvePasswordExpiration: () => ReactElement = (): ReactElement => {
+
+        return (
+            <>
+                <div className="title-header">
+                    <Heading as="h4">
+                        { t("validation:passwordExpiry.heading") }
+                    </Heading>
+                    <Heading as="h4">
+                        <Switch
+                            checked={ passwordExpiryEnabled }
+                            disabled={ isReadOnly }
+                            onChange={
+                                () => setPasswordExpiryEnabled(!passwordExpiryEnabled)
+                            } />
+                    </Heading>
+                </div>
+                <PasswordExpiryRuleList
+                    componentId={ "`${componentId}-password-expiry-rules`" }
+                    isPasswordExpiryEnabled={ passwordExpiryEnabled }
+                    isSkipFallbackEnabled={ passwordExpirySkipFallback }
+                    defaultPasswordExpiryTime={ defaultPasswordExpiryTime }
+                    ruleList={ passwordExpiryRules }
+                    rolesList={ allRoleList }
+                    groupsList={ groupsList.Resources }
+                    isReadOnly={ isReadOnly }
+                    onDefaultPasswordExpiryTimeChange={ (days: number) => setDefaultPasswordExpiryTime(days) }
+                    onSkipFallbackChange={ (skip: boolean) => setPasswordExpirySkipFallback(skip) }
+                    onRuleChange={ (newRuleList: PasswordExpiryRule[]) => setPasswordExpiryRules(newRuleList) }
+                    onRuleError={ (hasErrors: boolean) => setHasPasswordExpiryRuleErrors(hasErrors) }
+                />
+            </>
+        );
+    };
+
     const resolvePasswordValidation: () => ReactElement = (): ReactElement => {
         return (
             <div className="validation-configurations-form">
+                <Divider className="mt-4 mb-5" />
```

Comment:
refrain from using these kind of class names

Reply:
Removed